From d8fa572209f8de6c4a48dc8d3ab5eb5f4095c174 Mon Sep 17 00:00:00 2001
From: Keerthana K <keerthanak@vmware.com>
Date: Wed, 27 Jul 2022 11:40:49 +0000
Subject: [PATCH] setlocalversion: Skip reading localversion-rt file

This patch is required to skip reading localversion-rt file which
is appended to local version.

Signed-off-by: Keerthana K <keerthanak@vmware.com>
---
 scripts/setlocalversion | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/scripts/setlocalversion b/scripts/setlocalversion
index 2cb0b92f4..6ceb42b15 100755
--- a/scripts/setlocalversion
+++ b/scripts/setlocalversion
@@ -144,6 +144,9 @@ collect_files()
 		*\~*)
 			continue
 			;;
+		*localversion-rt)
+			continue
+			;;
 		esac
 		if test -e "$file"; then
 			res="$res$(cat "$file")"
-- 
2.28.0

